visallo/visallo
# convert resolving detected objects to use graph update context and have similar behavior to resolving term mentions (4.0)

Testing Instructions:

- Resolve a portion of the image with justification to a new entity
- open the new entity, make sure visibility "i" popover shows justification and default metadata

- Create a new entity with justification
- Resolve a portion of the image with justification to the entity you just created
- open the new entity, make sure visibility "i" popover shows original justification and default metadata

Points of Regression: Resolving portions of the image

CHANGELOG
Fixed: Resolving parts of an image didn't have all of the default metadata
